source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,284 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+In MongoDB, you can use a text search to retrieve documents that
+contain a term or a phrase in a specified field. A term is a sequence of
+characters that excludes whitespace characters. A phrase is a sequence
+of terms that with any number of whitespace characters.
```

Comment:
Nitpick:
I wonder if this is easier to read as separate paragraphs. Then you could list term and phrase with bold formatting so that the reader recognizes they're terms they should understand and remember.

Reply:
I tried it out and didn't think it was visually nice, so I kept it as it is.